lib/user.rb

```diff
@@ -0,0 +1,12 @@
+class User
+  attr_reader :id
+  attr_accessor :username, :password, :first_name, :last_name, :admin
+
+  def initialize(username, password, first_name, last_name = nil, admin = false)
+    self.username= username
+    self.password= password
+    self.first_name= first_name
+    self.last_name= last_name.empty? ? nil : last_name
```

Comment:
What if last_name comes as nil?

Reply:
You're correct that was an oversight on my side i've fixed this